Showcase-iOS/DependencyContainer/DependencyContainer.swift

```diff
@@ -102,6 +94,11 @@ struct DependencyContainer {
             return aboutPresenter
         }
         
+        container.register(AnalyticsManager.self) {r in
```

Comment:
Where is the interface that you are using to implement this analytics Manager? We need to have an interface with the methods on so that we can use it for mocking. Making a concrete version of this defeats the purpose of the dependancy container.

Reply:
Yeah, thanks.. I completely missed that... Done :)